source/reference/api/start.txt

```diff
@@ -350,6 +388,29 @@ Response
 Behavior
 --------
 
+.. _c2c-start-verifier:
+
+Embedded Verifier
+~~~~~~~~~~~~~~~~~
+
+Starting in 1.9, ``mongosync`` provides an embedded verifier to
+determine whether data transfer from the source cluster to the
+destination was successful.
+
+When enabled, the verifier performs a series of verification
+checks on the destination cluster. If any of these checks return
+an error, the verifier fails the migration. If all of the checks
```

Comment:
can we include an example of what such an error would look like? 

Also, we should clarify what guidance we will give to users that encounter errors in verification - presumably the only thing to do is restart mongosync from scratch. @mmcclimon is my understanding accurate?

Reply:
@mvankeulen94, I haven't seen any examples of what these errors look like. Given that you all are shipping soon I'd suggest we ticket the error example part out as a fast follow.

Reply:
I added this in the other thread but the error will include a message like `failed to finish commit: verifier indicated a mismatch between source & destination`

Reply:
Thinking some more, let's hold off on adding this example error